include/astshim/CmpMap.h

```diff
@@ -97,6 +97,10 @@ class CmpMap : public Mapping {
     bool getSeries() { return detail::isSeries(reinterpret_cast<AstCmpMap *>(getRawPtr())); }
 
 protected:
+    virtual std::shared_ptr<Object> copyPolymorphic() const override {
```

Comment:
Add a comment string saying what this is?

Reply:
Every single subclass of Object has one of these and it is documented there as something each subclass must override. I'm reluctant to add this to all subclasses on this ticket but will file a new one if you like. Doxygen already copies the docs.

Reply:
Actually I should mark these as `override`. I'll file a ticket.

Reply:
That's fine. Leave it.

Reply:
The reason I managed to miss this is that Mapping defines it. I wonder if I should eliminate that and `copy`, since Mapping is an abstract class.